### PR TITLE
Adding missing parameters

### PR DESCRIPTION
**Pull request by get-itips**
Closes https://github.com/MicrosoftDocs/office-docs-powershell/issues/6338

### CODE REVIEW

File: skype/skype-ps/skype/New-CsTenantNetworkSite.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ As an Admin, you can use the Windows PowerShell command, New-CsTenantNetworkSite
 ### Identity (Default)
 ```
 New-CsTenantNetworkSite [-Tenant <System.Guid>] [-Description <String>] [-NetworkRegionID <String>]
- [-LocationPolicy <String>] [-EnableLocationBasedRouting <Boolean>] [-OnlineVoiceRoutingPolicy <String>]
+ [-LocationPolicy <String>] [-EnableLocationBasedRouting <Boolean>] [-EmergencyCallRoutingPolicy <Object>] [-EmergencyCallingPolicy <Object>] [-OnlineVoiceRoutingPolicy <String>]
```
Comment (dariomws) [inline skype/skype-ps/skype/New-CsTenantNetworkSite.md:23]:
```suggestion
 [-LocationPolicy <String>] [-EnableLocationBasedRouting <Boolean>] [-EmergencyCallRoutingPolicy <String>] [-EmergencyCallingPolicy <String>] [-OnlineVoiceRoutingPolicy <String>]
```

Why not string? I think we always (or almost) use String for the policy name.

File: skype/skype-ps/skype/New-CsTenantNetworkSite.md [hunk 2]
```diff
@@ -105,6 +105,36 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -EmergencyCallRoutingPolicy
+This parameter is used to assign a custom emergency call routing policy to a network site. For more information see [Assign a custom emergency call routing policy to a network site](https://docs.microsoft.com/microsoftteams/manage-emergency-call-routing-policies#assign-a-custom-emergency-call-routing-policy-to-a-network-site).
+
+```yaml
+Type: Object
```
Comment (dariomws) [inline skype/skype-ps/skype/New-CsTenantNetworkSite.md:112]:
```suggestion
Type: String
```

File: skype/skype-ps/skype/New-CsTenantNetworkSite.md [hunk 3]
```diff
@@ -105,6 +105,36 @@ Accept pipeline input: False
...
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -EmergencyCallingPolicy
+This parameter is used to assign a custom emergency calling policy to a network site. For more information see [Assign a custom emergency calling policy to a network site](https://docs.microsoft.com/microsoftteams/manage-emergency-calling-policies#assign-a-custom-emergency-calling-policy-to-a-network-site).
+
+```yaml
+Type: Object
```
Comment (dariomws) [inline skype/skype-ps/skype/New-CsTenantNetworkSite.md:127]:
```suggestion
Type: String
```

File: skype/skype-ps/skype/New-CsTenantNetworkSite.md (hunk 1 above)
Comment (get-itips) [reply skype/skype-ps/skype/New-CsTenantNetworkSite.md]: PlatyPS put Object but is true that is better String here.

**Comment by yogkumgit** [comment]
@Kateyanne Please merge this PR. Thanks.

**Pull request merged**